orttraining/orttraining/test/python/orttraining_test_ortmodule_api.py

```diff
@@ -623,18 +640,27 @@ def test_input_requires_grad_backward_creates_input_grad_as_required0(device):
 
 
 
-@pytest.mark.parametrize("x1_requires_grad, x2_requires_grad", [(True, True), (True, False), (False, False), (False, True)])
-def test_input_requires_grad_backward_creates_input_grad_as_required1(x1_requires_grad, x2_requires_grad):
+@pytest.mark.parametrize("use_fp16, x1_requires_grad, x2_requires_grad", [
+    (False, True, True),
+    (False, True, False),
+    (False, False, False),
+    (False, False, True),
+    (True, True, True),
+    (True, True, False),
```

Comment:
May be replaced with:
@pytest.mark.parametrize("use_fp16", [True, False])
@pytest.mark.parametrize("x1_requires_grad", [True, False])
@pytest.mark.parametrize("x2_requires_grad", [True, False])